.editorconfig

```diff
@@ -0,0 +1,11 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
:thinking: never used this before. does it work with nvim? :)

Reply:
https://editorconfig.org/
> These editors come bundled with native support for EditorConfig. Everything should just work
> <image of Vim>